packages/redux/MIGRATION.md

```diff
@@ -590,24 +593,19 @@ The following tables contain the exports that were removed and the alternatives
 
 ##### Action types
 
-| Removed export                                | Notes                                |
-| --------------------------------------------- | ------------------------------------ |
-| actionTypes.POST_CHARGES\*¹                   | Removed as its action was deprecated |
-| actionTypes.COMPLETE_PAYMENT_CHECKOUT\*¹      | Removed as its action was deprecated |
-| actionTypes.COMPLETE_PAYMENT_CHECKOUT\*¹      | Removed as its action was deprecated |
-| actionTypes.UPDATE_GIFT_MESSAGE\*¹            | Removed as its action was deprecated |
-| actionTypes.UPDATE_DELIVERY_BUNDLE_UPGRADE\*¹ | Removed as its action was deprecated |
+| Removed export                     | Notes                                |
+| ---------------------------------- | ------------------------------------ |
+| actionTypes.POST_CHARGES\*¹        | Removed as its action was deprecated |
+| actionTypes.UPDATE_GIFT_MESSAGE\*¹ | Removed as its action was deprecated |
 
 ¹ Where `*` can be `_REQUEST`, `_FAILURE` or `_SUCCESS`.
 
 ##### Actions
 
-| Removed export                | Notes                                         |
-| ----------------------------- | --------------------------------------------- |
-| doCompletePaymentCheckout     | Removed as it was deprecated                  |
-| doPostCharges                 | Removed as it was deprecated                  |
-| doUpdateDeliveryBundleUpgrade | Removed as it was deprecated                  |
-| doUpdateGiftMessage           | Use `updateCheckoutOrderItems` action instead |
+| Removed export            | Notes                                         |
+| ------------------------- | --------------------------------------------- |
+| doCompletePaymentCheckout | Removed as it was deprecated                  |
```

Comment:
Since the `actionTypes.COMPLETE_PAYMENT_CHECKOUT` was renamed to CREATE_CHECKOUT_ORDER_CHARGE shouldn't this be added to the renamed section and point to the `createCheckoutOrderCharge` action as well?

Reply:
please see the response here: https://github.com/Farfetch/blackout/pull/800/files#r1245419787